include/dca/phys/dca_step/cluster_solver/ctaux/ctaux_walker.hpp

```diff
@@ -441,6 +441,7 @@ void CtauxWalker<device_t, parameters_type, MOMS_type>::initialize() {
 
 template <dca::linalg::DeviceType device_t, class parameters_type, class MOMS_type>
 void CtauxWalker<device_t, parameters_type, MOMS_type>::do_sweep() {
+  profiler_type profiler("do_sweep", "CT-AUX walker", __LINE__, thread_id);
```

Comment:
You added the profiler in do_sweep, but commented it out in many other places. 
Was that intentional, or just a case of forgetting yo remove the commented out ones before committing?

Reply:
No, it is intentional. This function is lengthy and I want it profiled.